### PR TITLE
Feature/crystal/loading fix

### PR DESCRIPTION
**Pull request by crystaltr**
## Proposed changes

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue(if any).

## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)
- [ ] DevOps related

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [project guidelines](https://github.com/Visual-Testing-for-Android-Apps/Project-guidelines) doc
- [x] I locally linted my code
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [x] Any dependent changes have been merged and published in downstream modules

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: src/components/reportPage/BatchJob.jsx [hunk 1]
```diff
@@ -51,9 +50,9 @@ const BatchJob = () => {
       />
 
       {isLoading && (
-        <div className="scrim">
+        <div class="loading style-2">
```
Comment (idomo1) [inline src/components/reportPage/BatchJob.jsx:53]: Should be `className`

File: src/components/reportPage/BatchJob.jsx [hunk 2]
```diff
@@ -51,9 +50,9 @@ const BatchJob = () => {
...
+          <div class="loading-wheel"></div>
```
Comment (idomo1) [inline src/components/reportPage/BatchJob.jsx:54]: Should be `className`

File: src/components/reportPage/batch-job.css [hunk 3]
```diff
@@ -48,17 +48,48 @@
...
+}
+
+.style-2 .loading-wheel {
+  border-style: solid;
+  border-color: #ccc transparent;
+}
+
+@keyframes spin {
+  0% {
+    -webkit-transform: rotate(0);
```
Comment (idomo1) [inline src/components/reportPage/batch-job.css:87]: Any reason you're using `-webkit-transform` over `transform`?

File: src/components/reportPage/batch-job.css [hunk 4]
```diff
@@ -48,17 +48,48 @@
...
+  }
+
+  100% {
+    -webkit-transform: rotate(-360deg);
```
Comment (idomo1) [inline src/components/reportPage/batch-job.css:91]: Same here

File: src/components/reportPage/batch-job.css [hunk 5]
```diff
@@ -48,17 +48,48 @@
...
+  margin-left: -40px;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  border-width: 30px;
+  border-radius: 50%;
+  animation: spin 1s linear infinite;
+}
+
+.style-2 .loading-wheel {
```
Comment (idomo1) [inline src/components/reportPage/batch-job.css:80]: Is there a `style-1`?

**Review by idomo1 — CHANGES_REQUESTED** [review]
Looks great 🥇

**Comment by mt-empty** [comment]
Could you please describe what `loading fix` means?

**Review by idomo1 — APPROVED** [review]

**Pull request merged**